### PR TITLE
Update user.rb

### LINKED ISSUE #1996: Add Email Validation
Hi, this is a first-timers-only issue. This means we've worked to make it more legible to folks who either haven't contributed to our codebase before, or even folks who haven't contributed to open source before.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you :gift_heart:

If you have contributed before, consider leaving this one for someone new, and looking through our general help wanted issues. Thanks!
:thinking: What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.
## Please describe the problem

A user is able to sign in and use publiclab with email wrong email addresses say bansal@gmail

:clipboard: Step by Step

:raising_hand_woman: Claim this issue: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

:memo: Add VALID_EMAIL_REGEX = /\A[\w+-.]+@[a-z\d-.]+.[a-z]+\z/i to the 
https://github.com/publiclab/plots2/blob/3b09a3515bbfecb48194bd86f2d467b2710103be/app/models/user.rb#L16
:floppy_disk: Commit your changes

:twisted_rightwards_arrows: Start a Pull Request.

:checkered_flag: Done Ask in comments for a review :)

:thinking::question: Questions?

Leave a comment below!

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. If someone seem stuck, offer them some help! Otherwise, take a look at some other issues you can help with. Thanks!
Thank you!

Your help makes Public Lab better! We deeply appreciate your helping refine and improve this site.

### PR DESCRIPTION
**Pull request by namanjain1812**
Closes #1996

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by SidharthBansal** [comment]
@namanjain1812 thanks for the pr. 
Can you please see the discussion which we had earlier regarding this issue in #2169? You can look at the files it may help.
 We are here to help you whenever you face any trouble ping any one of us.

File: app/models/user.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ class User < ActiveRecord::Base
 
   acts_as_authentic do |c|
     c.openid_required_fields = %i(nickname email)
+    VALID_EMAIL_REGEX = /\A[\w+-.]+@[a-z\d-.]+.[a-z]+\z/i
```
Comment (SidharthBansal) [inline app/models/user.rb]: Please add the regex expression into 
`` c.validates_format_of_email_field_options = { with: /@/ }``
also we had a discussion about regex in other pr, please have a look at it. 
Please refer rubular for regex.

**Comment by SidharthBansal** [comment]
@namanjain1812 oh I see you closed the pr. You can submit the changes into an existing branch. No need to close the pr and open a new one.

**Comment by namanjain1812** [comment]
am closing this pr and making a final new one. Thanks for the help @SidharthBansal i figured out what to do.
here is the new pr #2506

**Comment by SidharthBansal** [comment]
It would be great if we have one pr for one issue so that we can quickly see what discussion went previously. Hence, I will encourage you to make changes in the same pr.

**Pull request closed without merging**